### PR TITLE
Doc/mc 10204 documentation api

### PR DESCRIPTION
**Pull request by fyrnaga**
Documentation changes for the kubernetes plugin and gcp kubernetes.

### CODE REVIEW

**Comment by fyrnaga** [comment]
There is still some work to be done to debug the API within GCP.

**Comment by evanluc** [comment]
Looks like you did `git add .` :0

**Review by evanluc — APPROVED** [review]
LGTM. We should brainstorm a way to avoid needing to make two copies for the standalone and GCP. When Azure comes I think it will use the same format at GCP with the cluster id

File: source/includes/gcp/_k8_charts.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+      },
+      "insecureServer": false
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
```
Comment (franzpgarcia) [inline source/includes/gcp/_k8_charts.md:76]: that's a big example XD

File: source/includes/gcp/_k8_pods.md [hunk 2]
```diff
@@ -0,0 +1,918 @@
...
+        "qosClass": "Burstable",
+        "startTime": "2020-01-21T15:05:03.000-05:00"
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
```
Comment (franzpgarcia) [inline source/includes/gcp/_k8_pods.md:205]: I feel the examples are just too big. It will look weird. can we reduce it?

Comment (fyrnaga) [reply source/includes/gcp/_k8_pods.md:205]: These are simplified real example. We can talk on how we can make them shorter.

File: source/includes/gcp/_k8_charts.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/gcp/_k8_charts.md:76]: These are simplified real example. We can talk on how we can make them shorter.

File: source/includes/gcp/_k8_pods.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/gcp/_k8_pods.md:205]: fixed

File: source/includes/gcp/_k8_charts.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/gcp/_k8_charts.md:76]: fixed

**Review by franzpgarcia — APPROVED** [review]

**Pull request merged**